rel/i18n/emqx_bridge_azure_event_hub.hocon

```diff
@@ -262,29 +263,31 @@ kafka_message_value.label:
 """Message Value"""
 
 partition_strategy.desc:
-"""Partition strategy is to tell the producer how to dispatch messages to Azure Event Hubs partitions.
+"""Partition strategy is to tell the producer how to dispatch messages to partitions.
 
-<code>random</code>: Randomly pick a partition for each message
-<code>key_dispatch</code>: Hash Azure Event Hubs message key to a partition number"""
+<code>random</code>: Randomly pick a partition for each message.
+<code>key_dispatch</code>: Assigns messages to partitions based on a hash of the message key,
+ensuring consistent partition for messages with the same key."""
 
 partition_strategy.label:
 """Partition Strategy"""
 
 buffer_segment_bytes.desc:
 """Applicable when buffer mode is set to <code>disk</code> or <code>hybrid</code>.
-This value is to specify the size of each on-disk buffer file."""
+This setting specifies the size of each buffer file stored on disk."""
```

Comment:
```suggestion
This setting specifies the size of each buffer file segment stored on disk."""
```
?

Reply:
changed to 'segment fie', rather than 'file segment'